packages/cli/src/collaboration/collaboration.service.ts

```diff
@@ -73,34 +73,31 @@ export class CollaborationService {
 			return;
 		}
 
-		await this.state.removeActiveWorkflowUser(workflowId, userId);
+		await this.state.removeCollaborator(workflowId, userId);
 
 		await this.sendWorkflowUsersChangedMessage(workflowId);
 	}
 
 	private async sendWorkflowUsersChangedMessage(workflowId: Workflow['id']) {
 		// We have already validated that all active workflow users
 		// have proper access to the workflow, so we don't need to validate it again
-		const activeWorkflowUsers = await this.state.getActiveWorkflowUsers(workflowId);
-		const workflowUserIds = activeWorkflowUsers.map((user) => user.userId);
+		const collaborators = await this.state.getCollaborators(workflowId);
+		const userIds = collaborators.map((user) => user.userId);
 
-		if (workflowUserIds.length === 0) {
+		if (userIds.length === 0) {
 			return;
 		}
-		const users = await this.userRepository.getByIds(this.userRepository.manager, workflowUserIds);
-
-		const msgData: IActiveWorkflowUsersChanged = {
+		const users = await this.userRepository.getByIds(this.userRepository.manager, userIds);
+		const activeCollaborators = users.map((user) => ({
+			user: user.toIUser(),
```

Comment:
What's the difference between `IUser` and `Public user`? Which one should be used going forward?

Reply:
Maybe we could mark it as `@deprecated`